packages/typespec-ts/src/modular/helpers/operationHelpers.ts

```diff
@@ -1105,6 +1102,7 @@ function needsDeserialize(type?: Type) {
 
 export function hasLROOperation(codeModel: ModularCodeModel) {
   return (codeModel.clients ?? []).some((c) =>
+    c.rlcHelperDetails.hasLongRunning ||
```

Comment:
This is to resolve the LRO operation exists in RLC but is removed in the client decorator, and we have to use the core-lro dependency in package.json anyway.
@MaryGao, maybe you need to pay attention to other lro related logic in the new LRO implementation.